python_modules/dagster/dagster/_core/execution/context/output.py

```diff
@@ -113,7 +121,10 @@ def __init__(
         self._name = name
         self._job_name = job_name
         self._run_id = run_id
-        self._metadata = metadata or {}
+        normalized_metadata = normalize_renamed_param(
```

Comment:
the result of `normalize_renamed_param` is assigned to its own var first bc pyright was complaining if i had it as 

```python
self._definition_metadata = normalize_renamed_param(definition_metadata, "definition_metadata", metadata, "metadata") or {}
```